src/main/java/seedu/intrack/logic/commands/TaskCommand.java

```diff
@@ -0,0 +1,90 @@
+package seedu.intrack.logic.commands;
+
+import static seedu.intrack.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.intrack.model.Model.PREDICATE_SHOW_ALL_INTERNSHIPS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.intrack.commons.core.Messages;
+import seedu.intrack.commons.core.index.Index;
+import seedu.intrack.logic.commands.exceptions.CommandException;
+import seedu.intrack.model.Model;
+import seedu.intrack.model.internship.Internship;
+import seedu.intrack.model.internship.Task;
+
+/**
+ * Updates the current task of an Internship.
+ */
+public class TaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "task";
```

Comment:
Are we using `task` or `addtask`?

Reply:
Thank you for pointing that out, I have changed it to ```addtask``` to make it more clear that it is for adding a task.